tests/test_acl.py

```diff
@@ -560,9 +569,9 @@ def test_AclRuleIcmpV6(self, asic_db, config_db, state_db):
         self.remove_acl_table("test_icmpv6")
         self.verify_no_acl_tables()
 
-    def test_AclRuleRedirectToNextHop(self, dvs, asic_db, config_db, state_db):
+    def test_AclRuleRedirectToNextHop(self, dvs):
         dvs.setup_db()
```

Comment:
typo?

Reply:
`set_interface_status` uses `dvs.cdb` under the hood, so it is still needed for now. I can add a comment to make it more clear!